app/graphql/mutations/base_mutation.rb

```diff
@@ -62,6 +62,7 @@ def save_object(resource)
         errors = if resource.save
                    []
                  else
+                   resource.reload if resource.persisted?
```

Comment:
is hit the db necessary? maybe `restore_attributes` would be enough?
```suggestion
                   resource.restore_attributes if resource.persisted?
```

Reply:
`restore_attributes` would be enough, thanks.